satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -615,6 +615,31 @@ read all (
 	where storagenode_storage_tally.interval_end_time >= ?
 )
 
+//--- peer_identity ---//
+
+model peer_identity (
+    key serial_number 
+
+    index (
+        name serial_number_update_at
+        fields serial_number update_at 
+    )
+    
+    field serial_number  blob
+    field peer_identity  blob
```

Comment:
Let's rename this `der_chain` to be more descriptive how it's encoded.

Reply:
u meant `der_chain` or `peer_chain`

Reply:
I think `peer_` part is already implicit in the table name... however I guess `chain` would be fine as well. Except in that case add a comment about the encoding.

Reply:
changed to peer_chain and added a comment ` // x509 ASN.1 DER content `